secure/storage/src/on_disk.rs

```diff
@@ -55,14 +71,9 @@ impl OnDiskStorage {
         fs::rename(&self.temp_path, &self.file_path)?;
         Ok(())
     }
-
-    /// Public convenience function to return a new OnDiskStorage based Storage.
-    pub fn new_storage(path_buf: PathBuf) -> Box<dyn Storage> {
-        Box::new(OnDiskStorage::new(path_buf))
-    }
 }
 
-impl KVStorage for OnDiskStorage {
+impl<T: Send + Sync + TimeService> KVStorage for OnDiskStorageInternal<T> {
```

Comment:
from past experience, put Send + Sync in your trait saves more time typing

Reply:
My hope is to eliminate the Send + Sync at some point in the future... actually I wonder if it is even necessary any more...